### DISCUSSION TITLE
question/help - making all pages editable

### DISCUSSION THREAD

**Discussion by zsh28** [wagtail/wagtail#10581] [General]
I want to make my website customisable like all the pages from home page, about us, contact us etc are there any templates or tutorials on how best to do this.

**Comment by thibaudcolas** [comment]
👋 this is basically what Wagtail does! Have a look at our getting started tutorial if you haven’t already: https://docs.wagtail.org/en/stable/getting_started/tutorial.html.

**Reply by zsh28** [reply]
thank you for the reply I have been wondering how can I make my footer and navbar editaable point me in the right direction here if possibke also how can I implement bootstrap 5 for my website
